packages/teleport/src/Welcome/NewCredentials/NewCredentials.tsx

```diff
@@ -130,4 +136,5 @@ export function NewCredentials(props: State & Props) {
 
 export type Props = State & {
   resetMode?: boolean;
+  currFlow?: LoginFlow;
```

Comment:
Nit: maybe it would be more clear to name this `loginFlow`?

Reply:
I'm not going to rename it here because the[ origin ](https://github.com/gravitational/webapps/blob/b077b510f09b78b8d77fa39a9a43fbc813308488/packages/design/src/StepSlider/StepSlider.tsx#L25)is a few components up. I'm plumbing through one last child component. 

IMO, the pattern: `export type State = ReturnType<typeof useX>;` makes it hard to tell where a rename might impact other components